tests/integration/test_network_plugin.py

```diff
@@ -40,15 +39,33 @@ def setUp(self):
     def tearDown(self):
         super(NatRulesOperationsTestCase, self).tearDown()
 
-    def test_nat_rules_create_delete(self):
-        self.assertNotIn(self.public_ip, collectExternalIps(
-            self._get_gateway()))
+
+    def test_nat_rules_create_delete_with_explicit_ip(self):
+        self.ctx.node.properties['floatingip'].clear()
+        self.ctx.node.properties['floatingip'].update(IntegrationTestConfig().get()['floatingip'])
```

Comment:
Please add example configuration in examples/vcloud_integration_test_config.json.example